spec/requests/registration_number_forms_spec.rb

```diff
@@ -4,6 +4,10 @@
   include_examples "GET flexible form", form = "registration_number_form"
 
   describe "POST registration_number_forms_path" do
+    before do
```

Comment:
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀 
The stubbing makes so much sense when we aren't testing the CH service itself. This is awesome; I hope the effort is rewarded by some faster tests and not having to worry about valid CH numbers.
🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄🦄